src/SDK/Trace/SpanExporterInterface.php

```diff
@@ -12,16 +12,14 @@
  */
 interface SpanExporterInterface
 {
-    public static function fromConnectionString(string $endpointUrl, string $name, string $args);
```

Comment:
Should use a `SpanExporterFactoryInterface` instead to allow injecting dependencies.

Reply:
Seems like a reasonable thing to do.